docs/source/daml/intro/2_Scenario.rst

```diff
@@ -0,0 +1,139 @@
+
+Before you can create any ``Token`` contracts, you need some parties on the test ledger. In the above scenario, the function ``getParty`` is used to put a party called "Alice" in a valiable ``alice``. There are two things of note there:
+
+1. Note of the use of ``<-`` instead of ``=``. The reason for that is that depending on the ledger the scenario is running on, ``getParty`` may have to look up a party identity or create a new party. ``getParty`` is an ``Action`` that can only be performed once the ``Scenario`` is run in the context of a ledger and ``<-`` means "run the action and bind the result". More on ``Actions`` and ``do`` blocks in :doc:`5_Restrictions`.
+2. The argument ``"Alice"`` to ``getParty`` does not have to be enclosed in brackets. Functions in DAML are called using the syntax ``fn arg1 arg2 arg3``.
+
+With a variable ``alice`` of type ``Party`` in hand, you can submit your first transaction. Unsurprisingly, this is done using the ``submit`` function. ``submit`` takes two arguments: A Party and an ``Update``. Just like ``Scenario`` is a recipe for a test, ``Update`` is a recipe for a transaction. ``create Token with owner = alice`` is an ``Update``, which translates to the transaction creating a ``Token`` with owner Alice. You'll learn all about the syntax ``Token with owner = alice`` in :doc:`3_Data`.
+
+It's possible to write ``submit alice (create Token with owner = alice)``, but just like scenarios, updates can be assembled using ``do`` blocks. A ``do`` block always takes the value of the last statement within it so the syntax shown in the scenario above gives the same result, whilst being easier to read.
+
+Running Scenarios
+-----------------
+
+There are two ways to run scenarios:
+
+1. In the IDE, providing visualizations of the resulting ledger
+2. Using the command line, useful for continuous integration
+
+In the DAML IDE, you should see a small text reading "Scenario results" just above the line ``token_test_1 = do``. Click on it to display the outcome of the scenario.
+
+.. figure:: images/2_Scenario/scenario_results1.png
+
+This will open the scenario view in a separate column in VS Code. The default view is a tabular representation of the final state of the ledger:
+
+.. figure:: images/2_Scenario/tabular_view1.png
+
+The big title reading ``Intro_2_Scenario:Token`` is the identifier of the type of contract that's listed below. ``Intro_2_Scenario`` is the module name, ``Token`` the template name.
+
+The first columns, labelled vertically show which parties know about which contracts. In this simple scenario, the sole party "Alice" knows about the contract she created.
+
+The second column shows the status of the contract, either ``active`` or ``archived``. The remaining columns show the contract arguments, with one column per field. As expected, field ``owner`` is ``'Alice'``. The single quotation marks indicate that ``Alice`` is a party.
```

Comment:
The second column is actually the ID, which gets explained later on. I would say that explicitly here, along the lines of:
```suggestion
The second column shows the ID of the transaction. This will be explained later.

The third column shows the status of the contract, either ``active`` or ``archived``. The remaining columns show the contract arguments, with one column per field. As expected, field ``owner`` is ``'Alice'``. The single quotation marks indicate that ``Alice`` is a party.
```